sonar-db/src/main/java/org/sonar/db/version/v55/CreateInitialSchema.java

```diff
@@ -915,4 +915,8 @@ private void addIndex(Context context, String table, String index, boolean uniqu
     }
     context.execute(builder.build());
   }
+
+  private static VarcharColumnDef.Builder newVarcharBuilder(String column) {
```

Comment:
ok to rename but note that there won't be any future confusion as regular VARCHAR will never be used in this migration.